backend/app/api/endpoints/local_executor.py

```
@@ -0,0 +1,61 @@
+
+router = APIRouter()
+
+
+@router.post(
+    "/devices/{device_id}/capabilities/sync",
+    response_model=DeviceCapabilitySyncResponse,
+)
+async def sync_device_capabilities(
+    device_id: str,
+    request: DeviceCapabilitySyncRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)